Uchu.World/Packets/Server/ServerMailPacket.cs

```diff
@@ -111,13 +111,25 @@ public void Serialize(BitWriter writer)
 
                 foreach (var mail in Mails)
                 {
-                    var author = ctx.Characters.First(c => c.Id == mail.AuthorId);
                     
                     writer.Write(mail.Id);
 
                     writer.WriteString(mail.Subject, 50, true);
                     writer.WriteString(mail.Body, 400, true);
-                    writer.WriteString(author.Name, 32, true);
+
+                    string authorName;
+
+                    if (mail.AuthorId == 0)
+                    {
+                        authorName = "LEGO Universe";
```

Comment:
Maybe change this to Uchu?

Reply:
Went with this because that's how it was in live and the game is still called LEGO Universe, but if it's better to change it to Uchu that's fine with me. Just curious if you think this would be better out of concerns with TLG / usage of the LEGO trademark, or for clarity about whether the item comes from the server or from the client, or because of something else?

Reply:
For Uchu the server project it would most likely be fine to have "LEGO Universe" there. For Uchu the hosted project it would be better with an alternative. Maybe make if configurable.

Reply:
Oh great idea, I'll add a "Server name" to the configuration